.github/workflows/test-pr.yaml

```diff
@@ -67,7 +67,7 @@ jobs:
 
   test-scenarios:
     needs: [detect-changes]
-    runs-on: ubuntu-latest
+    runs-on: devcontainer-image-builder-ubuntu
```

Comment:
This seems to have fixed the space issue 🎉 

However, I am still seeing a failing container creation with error for `centos-7` 👇 

```
2024-02-07T18:43:25.1592288Z [2024-02-07T18:43:25.158Z] #16 103.0 checking for --with-openssl-rpath... configure: error: --with-openssl-rpath "/usr/local/python/3.11.8/lib" is not a directory
2024-02-07T18:43:25.1594634Z #16 103.1 ERROR: Feature "Python" (Unknown) failed to install! Look at the documentation at https://github.com/devcontainers/features/tree/main/src/python for help troubleshooting this error.
2024-02-07T18:43:25.1598300Z #16 ERROR: process "/bin/sh -c cp -ar /tmp/build-features-src/python_0 /tmp/dev-container-features  && chmod -R 0755 /tmp/dev-container-features/python_0  && cd /tmp/dev-container-features/python_0  && chmod +x ./devcontainer-features-install.sh  && ./devcontainer-features-install.sh  && rm -rf /tmp/dev-container-features/python_0" did not complete successfully: exit code: 1
2024-02-07T18:43:25.1600747Z 
2024-02-07T18:43:25.2094278Z [2024-02-07T18:43:25.209Z] ------
2024-02-07T18:43:25.2097638Z  > [dev_containers_target_stage 5/5] RUN --mount=type=bind,from=dev_containers_feature_content_source,source=python_0,target=/tmp/build-features-src/python_0     cp -ar /tmp/build-features-src/python_0 /tmp/dev-container-features  && chmod -R 0755 /tmp/dev-container-features/python_0  && cd /tmp/dev-container-features/python_0  && chmod +x ./devcontainer-features-install.sh  && ./devcontainer-features-install.sh  && rm -rf /tmp/dev-container-features/python_0:
2024-02-07T18:43:25.2100931Z 102.7 checking if the dirent structure of a d_type field... yes
2024-02-07T18:43:25.2101624Z 102.7 checking for the Linux getrandom() syscall... yes
2024-02-07T18:43:25.2102401Z 102.8 checking for the getrandom() function... no
2024-02-07T18:43:25.2103507Z 102.8 checking for library containing shm_open... -lrt
2024-02-07T18:43:25.2104240Z 102.9 checking for shm_open... yes
2024-02-07T18:43:25.2104685Z 102.9 checking for shm_unlink... yes
2024-02-07T18:43:25.2105141Z 103.0 checking for openssl/ssl.h in /usr/local/python/3.11.8... yes
2024-02-07T18:43:25.2105718Z 103.0 checking whether compiling and linking against OpenSSL works... yes
2024-02-07T18:43:25.2106680Z 103.0 checking for --with-openssl-rpath... configure: error: --with-openssl-rpath "/usr/local/python/3.11.8/lib" is not a directory
```